backend/utils/user_context.py

```diff
@@ -37,6 +36,11 @@ def get_organization() -> Optional[Organization]:
                 )
             except Organization.DoesNotExist:
                 return None
+            except ProgrammingError:
+                # Handle cases where the database schema might not be fully set up,
+                # especially during the execution of management commands
+                # other than runserver
```

Comment:
@muhammad-ali-e should we log something informative here?

Reply:
It should be a unenecessory logs during commands.